ui/src/app/views/workspace-edit/component.css

```diff
@@ -0,0 +1,22 @@
+textarea {
+  resize: none;
```

Comment:
Combine these two declarations into one `textarea`. Also, the padding around this makes me think I'm starting on the second line. Can we fix that?